src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/TemporaryAbsencesResource.kt

```diff
@@ -0,0 +1,74 @@
+package uk.gov.justice.digital.hmpps.welcometoprison.resources
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.welcometoprison.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.welcometoprison.model.TemporaryAbsence
+import uk.gov.justice.digital.hmpps.welcometoprison.model.prison.PrisonService
+
+@RestController
+@Validated
+@PreAuthorize("hasRole('SYSTEM_USER')")
+@RequestMapping(name = "Prison", path = ["/temporary-absences"], produces = [MediaType.APPLICATION_JSON_VALUE])
+class TemporaryAbsencesResource(
+  private val prisonService: PrisonService,
+) {
+  @Operation(
+    summary = "Retrieves the temporary absences for a prison",
+    description = "Retrieves the current temporary absences information, role required is SYSTEM_USER",
+    security = [SecurityRequirement(name = "SYSTEM_USER", scopes = ["read"])],
```

Comment:
Not absolutely sure whether access should be restricted the SYSTEM_USER role or indeed any role at all.